rllib/offline/estimators/doubly_robust.py

```diff
@@ -26,7 +26,7 @@
 
 @DeveloperAPI
 class DoublyRobust(OffPolicyEstimator):
-    """The Doubly Robust estimator.
```

Comment:
Wait, these aren't regexes

Reply:
Yes, but docstrings can also contain invalid escape sequences. See for instance the noise coming out of mypy in https://github.com/ray-project/ray/issues/48921, which points to malformed docstrings.

I would assume that most likely these docstrings were written thinking that a `\` is just a plain `\`, which raw string literals achieve.

Reply:
Right, but the solution isn't to annotate it as regex though, right?

Solution should be to escape the slash

Reply:
What do you mean by "annotate it as regex"?

Using [raw string literals](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals) like r"""...""" has nothing to do with regexes. They are just string literals where the `\` is not used as an escape code for introducing special chars like `\n` or `\r`.

It just happens to be common to use raw string literals when writing regexes, because it often involves `\` and the raw string literal avoid having to escape them all.